vnext/local-cli/generator-windows/templates/cs/proj/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.UI.Xaml" version="2.3.191129002" targetFramework="uap10.0.16299" />
```

Comment:
interestingly the .csproj uses version 255.255.255.255 that doesn't match... Is that expected?

Reply:
I also don't see Microsoft.UI.Xaml.Design which was references from the csproj

---
In reply to: [421615927](https://github.com/microsoft/react-native-windows/pull/4816#discussion_r421615927) [](ancestors = 421615927)

Reply:
it's a dll that's included in the mux package, and supports the designed in VS. it's a bug that it gets added when using packages.config, which is why I removed pacakges.config